<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoPlay</class>
 <widget class="QDialog" name="VideoPlay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>746</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Home&gt;Registration&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="subName">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="subMsg">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>321</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="videoWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>471</width>
     <height>192</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="lastVideo">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>390</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Play Last Video</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quizScreen">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>390</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Complete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="subjectScreen">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>390</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose a New Subject</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
